.github/workflows/tests.yml

```diff
@@ -7,51 +7,40 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, v2/dev ]
 
 jobs:
   build:
 
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]  # TODO: fix windows permissions issues and add windows-latest
-        python-version: [3.7, 3.8]
-        exclude:
-        # excludes node 8 on macOS
-          - os: macos-latest
-            python-version: 3.8
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.11]
 
     steps:
     - uses: actions/checkout@v2
     - uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: ${{ matrix.python-version }}
-        mamba-version: "*"
         environment-file: environment.yml
-        channels: conda-forge, defaults
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
         activate-environment: chemprop
-
-    - name: Set temp directories on Windows
-      shell: bash -l {0}
-      if: matrix.os == 'windows-latest'
-      run: |
-        echo "TMPDIR=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
-        echo "TEMP=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
-        echo "TMP=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
+        use-mamba: true
     - name: Install dependencies
       shell: bash -l {0}
       run: |
         mamba install flake8 pytest parameterized
-        python -m pip install -e .
-    - name: Lint with flake8
+        python -m pip install -e ".[dev,docs,test]"
+    - name: Test with pytest
+      shell: bash -l {0}
+      run: |
+        pytest -v chemprop/v2/tests
+    - name: Lint with flake8  #move this back before the tests after v2 development is done
```

Comment:
```suggestion
    - name: Lint with flake8  # TODO : move this back before the tests after v2 development is done
```

Reply:
Thanks for catching this, going to move to line above and add TODO